code/modules/clothing/head/helmet.dm

```diff
@@ -80,6 +77,12 @@
 	can_toggle = 1
 	toggle_cooldown = 20
 	activation_sound = 'sound/items/WEEOO1.ogg'
+	
+/obj/item/clothing/head/helmet/justice/attack_self()

```

Comment:
It was actually placed in clothing.dm for /obj/item/clothing. I searched through for activation_sound and that's the only other place in the code it appears.

So it's not so much a system right now as it is someone created a new var just for justice helmets and then added a check just in helmets for it.

I guess I could rename the  var/add the check though